.github/workflows/release.yml

```diff
@@ -0,0 +1,107 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to release"
+        required: true
+jobs:
+  release:
+    name: Release
+    runs-on: macOS-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Xcode 12.4
+        run: sudo xcode-select -switch /Applications/Xcode_12.4.app
+
+      # Check for unreleased section in changelog
```

Comment:
Nitpick: some of these comments seem helpful, but others are identical to the names - maybe we could make all the names explanatory and remove the extra comments?

Reply:
Good call -- we updated the names and removed the comments.